src/app/session/pages/login/login.component.scss

```diff
@@ -1,15 +1,50 @@
+:host {
+    background-color: #F9F9F9;
+    display: flex;
+    justify-content: center;
+    flex-basis: 100%;
+}
 
-form {
+.header {
+    height: 15%;
```

Comment:
Again. If you want top spacing, use `margin-top` or `padding-top` on another element.